content/blog/2020/04/2020-04-29-jenkins-is-the-way.adoc

```diff
@@ -0,0 +1,66 @@
+Then we wondered, why isn't anyone collecting these user stories and sharing them with the Jenkins community.
+
+== Introducing Jenkins is the Way
+
+So we took the first step to record and archive all the great stuff everyone in our community is building with Jenkins.
+This way, Jenkins users old and new can come to an archive and search for Jenkins solutions for inspiration.
+We foresee a vast library of solutions from all around the world, solving a wide array of challenges in every industry imaginable.
+We decided to call this archive "*Jenkins Is The Way*" and host it at https://JenkinsIsTheWay.io .
+
+To aggregate all these stories, we built a simple online questionnaire so that Jenkins users can submit their own experience using this leading open source automation server.
+With so many plugins to support building, deploying, and automating your projects, we expect to see a vast collection of stories.
+
+We've already received a handful, including stories that illustrate how *Jenkins Is The Way*:
+
+* to link:https://jenkinsistheway.io/user-story/jenkins-is-the-way-to-code-your-own-release-pipelines/[code your own release pipelines]
+* to link:https://jenkinsistheway.io/user-story/jenkins-is-the-way-to-cast-magic-of-continuous-delivery/[cast magic of continuous delivery]
+* to link:https://jenkinsistheway.io/user-story/to-understand-and-simplify-your-software-lifecycle/[understand and simplify your software lifecycle]
+* to link:https://jenkinsistheway.io/user-story/jenkins-is-the-way-to-accelerate-automation-in-the-cloud/[accelerate automation in the cloud]
+* to link:https://jenkinsistheway.io/user-story/jenkins-is-the-way-to-facilitate-day-to-day-work/[facilitate day-to-day work]
+
+== Add your story. Show your Jenkins pride. Get our T-shirt
+
+image:/images/post-images/jenkins-is-the-way/jenkins-is-the-way-t-shirt.png[Jenkins Is The Way T-shirt, role=center, float=left, height=229]
+
+Be an inspiration to the Jenkins community by sharing your Jenkins story.
+Just go to link:https://www.surveymonkey.com/r/JenkinsIsTheWay[this link and fill out the form].
+We'll ask you about your project's goals, the technical challenges you overcame with Jenkins, and the solutions you created.
+It should take no more than 20-30 minutes to complete.
+
+We'll clean it up for clarity and publish it on https://JenkinsIsTheWay.io .
+
+Once it's part of our archive, we'll send you our new 2020 *Jenkins Is the Way* t-shirt.
+
+And since the more, the merrier, please share this blog post with peers and colleagues.
+We want to hear everyone's stories about the clever ways Jenkins is used to automate all that we need to do.
+
+=== Thanks and Acknowledgement
+
+Special thanks to abConsulting for creating and managing the https://JenkinsIsTheWay.io site and for reviewing, editing, and publishing the submitted stories.
```

Comment:
link to abConsulting  ?

Reply:
Checking with them to see if they have a web presence.  I've not seen one previously.